src/services/EdFi.DataManagementService.Api.Tests/RestClient/test.http

```diff
@@ -0,0 +1,165 @@
+### Test POST of invalid resource
```

Comment:
Taken from Meadowlark.net spike

Reply:
What do you think about moving RestClient tests up much higher, maybe into `/tests`?

Reply:
Done